src/com/projectkorra/projectkorra/airbending/AirBlast.java

```diff
@@ -255,7 +257,7 @@ private void affect(final Entity entity) {
 			push.normalize().add(entity.getVelocity()).multiply(knockback);
 		}
 		
-		GeneralMethods.setVelocity(entity, push);
+		GeneralMethods.setVelocity(entity, push.normalize().multiply(knockback));

```

Comment:
`push.normalize().multiply(knockback)` takes place on line 254 of this file, that is why it was not present in this method call.

Reply:
I was making it match AirSuction in terms of how the velocity is applied, because if you look in AirSuction's progress its exactly the same as the above..

Reply:
It will always override the change the above if statement makes should it run is also redundant.